examples/deployment/kubernetes/README.md

```diff
@@ -50,12 +50,39 @@ curl -X POST ${LOG_URL}/v1beta1/logs/${tree_id}:init
 
 ```
 
-The easiest way to do this is probably to use `kubectl exec <name of one of the logserver pods> -ti -- /bin/bash` to get a shell on a logserver Pod, and use curl from there.
+The easiest way to do this is to port-forward one of the logserver pods to your local workstation `kubectl port-forward <name of one of the logserver pods> 8091:8091` and use curl to access the API endpoint:
 
-(Use `kubectl get pods` to retrieve a list of all the Pods.)
+1. Grab the name of 0th logserver pod:
+```
+LOG_POD=$(\
+  kubectl get pods \
+  --selector=io.kompose.service=trillian-log \
+  --output=jsonpath='{.items[0].metadata.name}')
+```
+1. Port forward this pod's port `8091` to your local workstation (we're using `8091` locally too):
+```
+kubectl port-forward ${LOG_POD} 8091:8091
+```
+1. Create a Trillian tree:
+```
+RESPONSE=$(curl \
+--silent \
+--request POST \
+http://localhost:8091/v1beta1/trees \
+--data '{ "tree":{ "tree_state":"ACTIVE", "tree_type":"LOG", "hash_strategy":"RFC6962_SHA256", "signature_algorithm":"ECDSA", "max_root_duration":"0", "hash_algorithm":"SHA256" }, "key_spec":{ "ecdsa_params":{ "curve":"P256" } } }')
+TREE_ID=$(echo ${RESPONSE} | jq --raw-output .tree_id)
+echo ${RESPONSE}
+
+```
+1. Initialize it:
+```
+curl \
+--request POST \
+http://localhost:8091/v1beta1/logs/${TREE_ID}:init
+```
 
-**NOTE: none of the Trillian APIs are exposed to the internet with this config,
-this is intentional since the only access to Trillian should be via a
+**NOTE: none of the Trillian APIs is exposed to the Internet with this config.
```

Comment:
This scans weirdly now because "APIs" is plural, did you mean to change are -> is/did you want to change APIs -> API too?

Reply:
None of the cool kids capitalise `internet` [these days](https://en.wikipedia.org/wiki/Capitalization_of_%22Internet%22#Usage_examples) :)

Reply:
Ack... 

Let's revert to "none of the Trillians APIS are"

I'll never yield to "on-premise" as a neologism for "on-premises" but happy (thanks!) to be upgraded on "internet". Does this mean it's not "the internet" anymore either then? I'm so old (school).

Reply:
Oke doke :)

I'm old school too, and will stand shoulder to shoulder with you against "on-premise" _on the premise_ that it's Plain Wrong.

FWIW, personally I say "the internet" too, although interestingly enough in peninsular Spanish it seems to be just "internet" rather than "el internet". Go figure :)